e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -492,26 +492,19 @@ describeEE("scenarios > embedding > full app", () => {
           },
         });
       });
-      cy.get("@postMessage")
-        .should("have.been.calledWith", {
-          metabase: {
-            type: "frame",
-            frame: {
-              mode: "fit",
-              height: Cypress.sinon.match(value => value > 1000),
-            },
-          },
-        })
-        .and("not.have.been.calledWith", {
-          metabase: {
-            type: "frame",
-            frame: {
-              mode: "fit",
-              height: Cypress.sinon.match(value => value < 400),
-            },
+
+      // TODO: Find a way to assert that this is the last call.
+      cy.get("@postMessage").should("have.been.calledWith", {
+        metabase: {
+          type: "frame",
+          frame: {
+            mode: "fit",
+            height: Cypress.sinon.match(value => value > 1000),
           },
-        });
+        },
+      });
 
+      cy.get("@postMessage").invoke("resetHistory");
```

Comment:
Reset the mock before performing an action that would change the dashboard height. This way we can be confident that the call from previous steps won't interfere with the assertion.

I've verified that this does work by testing that the previous call (height > 1000px) won't be called.